modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -217,10 +219,10 @@ protected function register_controls() {
 				],
 			],
 			'selectors_dictionary' => [
-				'start' => '--n-tabs-heading-justify-content: flex-start; --n-tabs-title-width: initial; --n-tabs-title-height: initial; --n-tabs-title-align-items: center; --n-tabs-title-flex-grow: 0;',
-				'center' => '--n-tabs-heading-justify-content: center; --n-tabs-title-width: initial; --n-tabs-title-height: initial; --n-tabs-title-align-items: center; --n-tabs-title-flex-grow: 0;',
-				'end' => '--n-tabs-heading-justify-content: flex-end; --n-tabs-title-width: initial; --n-tabs-title-height: initial; --n-tabs-title-align-items: center; --n-tabs-title-flex-grow: 0;',
-				'stretch' => '--n-tabs-heading-justify-content: initial; --n-tabs-title-width: 100%; --n-tabs-title-height: initial; --n-tabs-title-align-items: center; --n-tabs-title-flex-grow: 1;',
+				'start' => '--n-tabs-justify-content: flex-start; --n-tabs-title-align-items: center; --n-tabs-title-flex-grow: 0; --n-tabs-spacer-top-height: 0px; --n-tabs-spacer-bottom-height: 0px;',
+				'center' => '--n-tabs-justify-content: center; --n-tabs-title-align-items: center; --n-tabs-title-flex-grow: 0; --n-tabs-spacer-top-height: 0px; --n-tabs-spacer-bottom-height: 0px;',
+				'end' => '--n-tabs-justify-content: flex-end; --n-tabs-title-align-items: center; --n-tabs-title-flex-grow: 0; --n-tabs-spacer-top-height: 0px; --n-tabs-spacer-bottom-height: 0px;',
+				'stretch' => '--n-tabs-justify-content: initial; --n-tabs-title-align-items: center; --n-tabs-title-flex-grow: 1; --n-tabs-spacer-top-height: 0px; --n-tabs-spacer-bottom-height: 0px;',
```

Comment:
Same here, prefer logical properties over physical properties when naming CSS variables.

For example, replace `--n-tabs-spacer-top-height` with `--n-tabs-spacer-block-start-height`.